ledger/participant-state/kvutils/src/main/protobuf/com/daml/ledger/participant/state/kvutils/daml_kvutils.proto

```diff
@@ -469,6 +490,8 @@ message DamlContractKeyState {
 
 // Errors
 
+// V1
```

Comment:
If you want to mark a block of messages belonging to a version, please either create a multi-line header such as the below:
```protobuf
//
// Used by participant.state.v1 API.
//
```
or add a comment e.g. `// Used by participant.state.v1 API.` to each message.